internal/kafka/test/integration/kafka_create_test.go

```diff
@@ -580,7 +582,7 @@ func TestKafkaCreate_ValidatePlanParam(t *testing.T) {
 	g.Expect(err).ToNot(gomega.HaveOccurred())
 	instanceTypeSizeConfig, err := instanceTypeConfig.GetKafkaInstanceSizeByID("x1")
 	g.Expect(err).ToNot(gomega.HaveOccurred())
-	g.Expect(*kafka.ExpiresAt).To(gomega.Equal(kafka.CreatedAt.Add(time.Duration(*instanceTypeSizeConfig.LifespanSeconds) * time.Second)))
+	g.Expect(kafka.ExpiresAt.Equal(kafka.CreatedAt.Add(time.Duration(*instanceTypeSizeConfig.LifespanSeconds) * time.Second))).To(gomega.BeTrue())
```

Comment:
@JameelB @machi1990 - the changed line started to fail for me locally since this morning with the error:

```
kas-fleetshard-sync.go:215: Starting mock fleetshard sync
I1028 10:57:32.761929   57616 kafkas_routes_cname_mgr.go:48] kafkas need routes created count = 0
    kafka_create_test.go:586: 
        Expected
            <time.Time>: 2022-10-30T09:57:32.810862Z
        to be equivalent to
            <time.Time>: 2022-10-30T09:57:32.810862Z
```

see [this](https://stackoverflow.com/questions/69350229/go-assert-equal-some-interface-with-time-time-object-inside) for why it might happen.